site/content/blogs/release-v119.en.md

```diff
@@ -42,7 +42,7 @@ http:
       hosted_zone: HostedZoneID2
 ```
 * **Access to Created Private Route Tables**:
-Copilot now exports private route table IDs from CloudFormation environment stacks. Use them to create VPC gateway endpoints with [addons](../docs/developing/addons/modeling.en.md). ([#3611](https://github.com/aws/copilot-cli/pull/3611))
+Copilot now exports private route table IDs from CloudFormation environment stacks. Use them to create VPC gateway endpoints with [addons](../../docs/developing/addons/workload). ([#3611](https://github.com/aws/copilot-cli/pull/3611))
```

Comment:
Do we not need the en.md suffix?

Reply:
I don't think so! Not according to my local site brought up by `make local-site`